controllers/comments-controller.js

```diff
@@ -14,5 +17,16 @@ exports.getCommentsByReviewId = (req, res, next) => {
     .catch((err) => {
       next(err);
     });
+};
 
+exports.postCommentOnReview = (req, res, next) => {
+  const { review_id } = req.params;
+  const content = req.body;
+  addCommentOnReview(review_id, content)
+    .then((addedComment) => {
+      res.status(201).send({ addedComment });

```

Comment:
The response body should match the convention for returning comments, in this case {comment : {...} }